reference/shaders-hlsl/vert/basic.vert

```diff
@@ -0,0 +1,47 @@
+struct _UBO
+{
+    float4x4 uMVP;
+};
+
+cbuffer UBO
+{
+    _UBO _16;
```

Comment:
Is there a reason the UBO block data is not inlined directly into the cbuffer? Is it possible to declare the cbuffer as

```
cbuffer UBO { float4x4 uMVP } _16;
```
instead?